packages/browser-sdk/src/lib/react/useRoomConnection/types.ts

```diff
@@ -44,14 +45,15 @@ export type LiveStreamState = {
 };
 
 export interface RoomConnectionState {
+    connectionStatus: ConnectionStatus;
     chatMessages: ChatMessage[];
     cloudRecording?: CloudRecordingState;
+    events?: EventEmitter;
```

Comment:
`EventEmitter` is a generic now. Would be nice to have the generic type specified. You can look for some exmaples  [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68313/files#diff-8a327f17520cd48deb2e32015ac880228b1ca388507256de6021a8f808a08335) in the tests.

Reply:
It was a bit of work but I managed to add the types in https://github.com/whereby/sdk/pull/244/commits/600901e8c43fcd8b667e1f3866f09c1a1f5862e5.